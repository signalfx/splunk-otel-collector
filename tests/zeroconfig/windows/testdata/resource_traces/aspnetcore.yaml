resource_spans:
- attributes:
    deployment.environment: zc-iis-test
    host.name: <ANY>
    os.type: windows
    process.pid: <ANY>
    process.runtime.description: <ANY>
    process.runtime.name: .NET
    process.runtime.version: <ANY>
    service.name: AspNetCore.WebApi.Net
    splunk.distro.version: SPLUNK_OTEL_DOTNET_VERSION
    splunk.zc.method: splunk-otel-dotnet-vSPLUNK_OTEL_DOTNET_VERSION
    telemetry.distro.name: splunk-otel-dotnet
    telemetry.distro.version: SPLUNK_OTEL_DOTNET_VERSION
    telemetry.sdk.language: dotnet
    telemetry.sdk.name: opentelemetry
    telemetry.sdk.version: <ANY>
  scope_spans:
    - instrumentation_scope:
        attributes: {}
        name: OpenTelemetry.Instrumentation.AspNetCore
        version: <ANY>
      spans:
        - attributes:
            http.request.method: GET
            http.response.status_code: 200
            http.route: /api/values/{id}
            network.protocol.version: "1.1"
            server.address: localhost
            server.port: 8000
            url.path: /aspnetcoreapp/api/values/6
            url.scheme: http
            user_agent.original: Go-http-client/1.1
          name: GET /api/values/{id}
