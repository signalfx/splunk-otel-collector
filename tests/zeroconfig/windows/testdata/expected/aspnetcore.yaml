resourceSpans:
  - resource:
      attributes:
        - key: service.name
          value:
            stringValue: AspNetCore.WebApi.Net
        - key: splunk.distro.version
          value:
            stringValue: 1.7.0
        - key: telemetry.distro.name
          value:
            stringValue: splunk-otel-dotnet
        - key: telemetry.distro.version
          value:
            stringValue: 1.7.0
        - key: os.type
          value:
            stringValue: windows
        - key: os.description
          value:
            stringValue: Microsoft Windows 10.0.20348
        - key: os.build_id
          value:
            stringValue: "20348"
        - key: os.name
          value:
            stringValue: Windows Server 2022 Datacenter
        - key: os.version
          value:
            stringValue: "6.3"
        - key: host.name
          value:
            stringValue: e137fad28e80.nxjcjlpip0jelm5h5wo3zvqkif.dx.internal.cloudapp.net
        - key: host.id
          value:
            stringValue: e5d95f12-d17e-4836-acdb-af990a55bf63
        - key: process.owner
          value:
            stringValue: AspNetCoreAppPool
        - key: process.pid
          value:
            intValue: "2144"
        - key: process.runtime.description
          value:
            stringValue: .NET 6.0.33
        - key: process.runtime.name
          value:
            stringValue: .NET
        - key: process.runtime.version
          value:
            stringValue: 6.0.33
        - key: telemetry.sdk.name
          value:
            stringValue: opentelemetry
        - key: telemetry.sdk.language
          value:
            stringValue: dotnet
        - key: telemetry.sdk.version
          value:
            stringValue: 1.9.0
        - key: deployment.environment
          value:
            stringValue: zc-iis-test
        - key: splunk.zc.method
          value:
            stringValue: splunk-otel-dotnet-v1.8.0
    scopeSpans:
      - scope:
          name: OpenTelemetry.Instrumentation.AspNetCore
          version: 1.9.0.42
        spans:
          - attributes:
              - key: server.address
                value:
                  stringValue: localhost
              - key: server.port
                value:
                  intValue: "8000"
              - key: http.request.method
                value:
                  stringValue: GET
              - key: url.scheme
                value:
                  stringValue: http
              - key: url.path
                value:
                  stringValue: /aspnetcoreapp/api/values/6
              - key: network.protocol.version
                value:
                  stringValue: "1.1"
              - key: user_agent.original
                value:
                  stringValue: Go-http-client/1.1
              - key: http.route
                value:
                  stringValue: /api/values/{id}
              - key: http.response.status_code
                value:
                  intValue: "200"
            endTimeUnixNano: "1728082126065204500"
            kind: 2
            name: GET /api/values/{id}
            parentSpanId: ""
            spanId: 99a75dd0cb9dee6e
            startTimeUnixNano: "1728082125877857000"
            status: {}
            traceId: 62abd1bb6704e2d47f4c3875ed45f299